lms/envs/common.py

```diff
@@ -4883,6 +4887,10 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 BLOCKSTORE_PUBLIC_URL_ROOT = 'http://localhost:18250'
 BLOCKSTORE_API_URL = 'http://localhost:18250/api/v1/'
 
+# Disable the Blockstore app API by default.
+# See openedx.core.lib.blockstore_api.config for details.
+BLOCKSTORE_USE_BLOCKSTORE_APP_API = False
```

Comment:
Getting another couple of errors when I try step 4 of the instructions and enable the waffle flag.

When trying to view an existing image xblock created before turning on the waffle flag, the request is sent to studio  `http://edx.devstack.studio:18010/api/xblock/v2/xblocks/lb:LabXchange:7af1be60:lx_image:1/handler/8-29d99e752d1fc9783f51/v1_student_view_data/`. This results in the following error:

```
labxchange.service_clients.edxapp.client] client.py:226 - 500 error from http://edx.devstack.studio:18010/api/xblock/v2/xblocks/lb:LabXchange:7af1be60:lx_image:1/handler/8-29d99e752d1fc9783f51/v1_student_view_data/
```

Going via the LMS `http://localhost:18000/api/xblock/v2/xblocks/lb:LabXchange:7af1be60:lx_image:1/handler/8-29d99e752d1fc9783f51/v1_student_view_data/` in the browser does work however.

When creating a new xblock with the waffle flag turned on, I am getting:
```
[labxchange.service_clients.edxapp.client] client.py:226 - 500 error from http://edx.devstack.lms:18000/api/xblock/v2/xblocks/lb:LabXchange:7af1be60:lx_image:1/view/student_view/
```

Haven't been able to figure out what am I doing wrong. This seems to be a config issue. Have any tips?

Reply:
I'd need a traceback to know for sure, but those errors could be related to an outdated blockstore package. [As noted above](https://github.com/openedx/edx-platform/pull/29779#discussion_r812520392), could you fetch the latest version of https://github.com/openedx/blockstore/pull/149 and install it on your LMS/Studio, and try again?

Reply:
Finally tracked this down. Because the `private.py` files are loaded right at the end, `BUNDLE_ASSET_STORAGE_SETTINGS` gets configured with the default value `/edx/var/edxapp/media`. Need to add the following to `private.py` as well:

```
BUNDLE_ASSET_STORAGE_SETTINGS = dict(
    STORAGE_CLASS='django.core.files.storage.FileSystemStorage',
    STORAGE_KWARGS=dict(
        location='/edx/src/blockstore/blockstore/media',
        base_url='/media/',
    ),
)```

Reply:
Ah thank you! I've added this to the PR description.